airflow/providers/yandex/operators/yandexcloud_yq.py

```diff
@@ -0,0 +1,89 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, Sequence
+
+from airflow.providers.common.sql.operators.sql import SQLExecuteQueryOperator
+from airflow.providers.yandex.hooks.yq import YQHook
+from airflow.providers.yandex.links.yq import YQLink
+
+if TYPE_CHECKING:
+    from airflow.utils.context import Context
+
+
+class YQExecuteQueryOperator(SQLExecuteQueryOperator):
```

Comment:
Is it required to extend `SQLExecuteQueryOperator`? asked because this generic class for iterate around Hooks which implements (or thought that it implements) DBApi2.

If not just better to use BaseOperator instead

Reply:
> We use it for query text templating and for syntax highlighting (sql dialect). Can I achieve this with BaseOperator

1. `template_fields: Sequence[str] = ("sql",)` - uses for template operator's sql attribute
2. `template_ext: Sequence[str] = (".sql",)` - uses for template fields from the `.sql` files
3. `template_fields_renderers = {"sql": "sql"}` - uses for highlight syntax

Reply:
BaseOperator is much better!=) Thank you for help!